noxfile.py

```diff
@@ -158,19 +151,17 @@ def parser() -> ArgumentParser:
         )
         return p
 
-    with odbcconfig(ODBC_DRIVER) as (config, env):
-        args = parser().parse_args(session.posargs)
-        connector = args.connector
-        session.run(
-            *_coverage_command(),
-            "pytest",
-            "--dropfirst",
-            "--db",
-            f"exasol-{connector}",
-            f"{PROJECT_ROOT / 'test' / 'integration' / 'exasol'}",
-            external=True,
-            env=env,
-        )
+    args = parser().parse_args(session.posargs)
+    connector = args.connector
```

Comment:
There is now only one connector available. Does it make sense to pass this parameter from the nox config to the GH workflow to this nox task?

Reply:
It is possible that we get new connectors.